.travis.yml

```diff
@@ -0,0 +1,64 @@
+#
+# Turn off email notifications entirely
+#
+notifications:
+  email: false
+
+language: generic
+
+#
+# Define the build matrix
+#
+matrix:
+  include:
+#    - services: docker
+#      env: OS_RELEASE=debian_unstable
+#      sudo: required
+#      script:
+#        - docker-compose -f scripts/docker/docker-compose.yml build $OS_RELEASE
+#        - docker-compose -f scripts/docker/docker-compose.yml run $OS_RELEASE /bin/bash /workspace/code/scripts/travis.sh
+    - os: osx
+
+osx_image: xcode7.3 # use latest OS X
```

Comment:
Sure we can build on all available versions on travis once we get it to work at least on the current version.